server/lib/nl_page_config.py

```diff
@@ -154,6 +157,42 @@ def _multiple_place_bar_block(places: List[Place], svs: List[str], sv2name):
   return block, stat_var_spec_map
 
 
+def _topic_sv_blocks(category: subject_page_pb2.Category, topic_svs: List[str],
+                     extended_sv_map: Dict[str,
+                                           List[str]], sv2name, sv_exists_list):
+  """Fill in category if there is a topic."""
+  main_block = category.blocks.add()
+  column = main_block.columns.add()
+  for sv in topic_svs:
+    if 'dc/svpg/' in sv:
+      sub_svs = extended_sv_map[sv]
+      if not sub_svs:
+        continue
+      sub_svs_exist = filter(lambda x: x in sv_exists_list, sub_svs)
+      if not sub_svs_exist:
+        continue
+      # add a block for each peer group
+      block = category.blocks.add()
+      column = block.columns.add()
+      for i, sub_sv in enumerate(sub_svs_exist):
+        # split up into several line charts
+        if i % 5 == 0:
+          tile = column.tiles.add()
+          tile.type = subject_page_pb2.Tile.TileType.LINE
+          tile.title = nl_topic.svpg_name(sv)
+        tile.stat_var_key.append(sub_sv)
```

Comment:
nit: for future, for dc/svpg/AmountOfFarmInventoryByType, we might need unit handling (or more dc/svpg's).